addons/binding/org.openhab.binding.helios/src/main/java/org/openhab/binding/helios/handler/HeliosHandler213.java

```diff
@@ -0,0 +1,962 @@
+            if (jsonObject.get("success").toString().equals("true")) {
+                logger.info("Successfully requested switch capabilities from the Helios IP Vario '{}'",
+                        getThing().getUID().toString());
+                String result = jsonObject.get("result").toString();
+                result = result.replace("switch", "id");
+                JsonObject js = parser.parse(result).getAsJsonObject();
+                RESTSwitch[] switchArray = gson.fromJson(js.getAsJsonArray("ides"), RESTSwitch[].class);
+                if (switchArray != null) {
+                    return Arrays.asList(switchArray);
+                }
+            } else {
+                RESTError error = gson.fromJson(jsonObject.get("error").toString(), RESTError.class);
+                logger.error(
+                        "An error occurred while communicating with the Helios IP Vario '{}' : code '{}', param '{}' : '{}'",
+                        new Object[] { getThing().getUID().toString(), error.code, error.param, error.description });
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR);
+                scheduler.schedule(resetRunnable, RESET_INTERVAL, TimeUnit.SECONDS);
+                return null;
+            }
+        } else {
+            logger.warn("An error occurred while requesting switch capabilities from the Helios IP Vario '{}'",
+                    getThing().getUID().toString());
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR);
+            scheduler.schedule(resetRunnable, RESET_INTERVAL, TimeUnit.SECONDS);
+            return null;
+        }
+
+        return null;
+    }
+
+    private void triggerSwitch(String id) {
+        if (getThing().getStatus() == ThingStatus.ONLINE) {
+
+            switchTarget = baseTarget.path(SWITCH_PATH);
+
+            Response response = null;
+            try {
+                response = switchTarget.resolveTemplate("ip", ipAddress).resolveTemplate("cmd", CONTROL)
+                        .queryParam("switch", id).queryParam("action", "trigger")
+                        .request(MediaType.APPLICATION_JSON_TYPE).get();
+            } catch (Exception e) {
+                logger.error("An exception occurred while triggering a switch  on the Helios IP Vario '{}' : '{}'",
+                        getThing().getUID().toString(), e.getMessage());
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR);
+                scheduler.schedule(resetRunnable, RESET_INTERVAL, TimeUnit.SECONDS);
+                return;
+            }
+
+            if (response != null) {
+                JsonObject jsonObject = parser.parse(response.readEntity(String.class)).getAsJsonObject();
+                if (jsonObject.get("success").toString().equals("true")) {
+                    logger.info("Successfully triggered a switch on the Helios IP Vario '{}'",
```

Comment:
Please change the logging level to `debug`. `trace` can be used for even more details, where necessary.